server/app/routes/reviews.js

```diff
@@ -0,0 +1,42 @@
+var router = require('express').Router();
+
+var models = require('../db/models');
+
+var Review= models.Review;
+
+module.exports = router;
+
+router.param('reviewId', function(req, res,next,id){
+	Review.findById(id)
+	.then(review => req.review= review);
+});
+
+router.get('/', function (req, res, next) {
+  Review.findAll()
+  .then(reviews => res.json(reviews))
+  .catch(next);
+});
+
+
+router.get('/:reviewId', function (req, res, next) {
+  res.send(req.review);
```

Comment:
I think it looks "minified" when you res.send it vs res.json.  It probably
doesn't matter from a practical matter, but maybe just for consistency it
would be good to json.  But you can leave it as is for now.
ᐧ

On Mon, Jun 6, 2016 at 10:14 PM, ARRiera96 <notifications@github.com> wrote:

> In server/app/routes/reviews.js
> <https://github.com/jdhang/pandoras-kits/pull/11#discussion_r66000750>:
>
> > +module.exports = router;
> > +
> > +router.param('reviewId', function(req, res,next,id){
> > +	Review.findById(id)
> > +	.then(review => req.review= review);
> > +});
> > +
> > +router.get('/', function (req, res, next) {
> > +  Review.findAll()
> > +  .then(reviews => res.json(reviews))
> > +  .catch(next);
> > +});
> > +
> > +
> > +router.get('/:reviewId', function (req, res, next) {
> > +  res.send(req.review);
>
> I think res.send will send it as json anyways. You guys know the benefit
> of one over the other?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/jdhang/pandoras-kits/pull/11/files/a70b862df10937068bdf28c7c2922773c598cf9f#r66000750>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AO_ujUoSyc_5UggjfGwwhWUU0DRY52pwks5qJNQBgaJpZM4IvRlp>
> .
>